cmd/deploy/deploy.go

```diff
@@ -346,6 +350,13 @@ func getProxyHandler(name, token string, clusterConfig *rest.Config) (http.Handl
 			}
 			metadata.Labels[model.DeployedByLabel] = name
 
+			if metadata.Annotations == nil {
+				metadata.Annotations = map[string]string{}
+			}
+			if isOktetoRepo {
```

Comment:
This is not safe to use, at least by convention. It's technically probably fine but it's bad practice to use any other return value from a function if the err is non-nil.

Best practice would be to explicitly set `isOktetoRepo` to `false` in the error handling.